prometheus-exporters/esxi-exporter/templates/deployment.yaml

```diff
@@ -6,31 +8,34 @@ metadata:
 
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: esxi-exporter-{{ required "$collector is missing" $collector }}
+      type: exporter
   template:
     metadata:
       labels:
-        app: esxi-exporter
+        app: esxi-exporter-{{ required "$collector is missing" $collector }}
         type: exporter
-      annotations:
-        prometheus.io/scrape: "true"
-        prometheus.io/port: {{ required ".Values.listen_port missing" .Values.listen_port | quote }}
-        prometheus.io/targets: {{ required ".Values.alerts.prometheus missing" .Values.alerts.prometheus | quote }}
```

Comment:
How are you scraping now? Don't see any SD configuration change in this PR

Reply:
We decided to use kubernetes SD. So no default config is necessary.

Reply:
kubernetes or prometheus?

Reply:
we plan to use the kubernetes sd for prometheus like the vrops exporter to set variable scrape times.